samples/net/wpan_serial/prj.conf

```diff
@@ -9,7 +9,6 @@ CONFIG_UART_LINE_CTRL=y
 
```

Comment:
Commit message:

~~~
 samples: wpan_serial: Allow to use other 15.4 devices

This allows to select IEEE802.15.4 chips.
~~~

The 2nd sentence looks strange. Did you miss word "other"? Having something like "This allows to select different IEEE802.15.4 chips instead of hardcoded cc2520" would be just the clean and clear explanation of what this change does.